manifests/mod/alias.pp

```diff
@@ -1,7 +1,7 @@
 class apache::mod::alias(
   $apache_version = $apache::apache_version
 ) {
-  $ver24 = versioncmp($apache_version, 2.4) >= 0
+  $ver24 = versioncmp("$apache_version", "2.4") >= 0
```

Comment:
For consistency with the other places this is used, could you change "2.4" to '2.4' and drop the quotes around $apache_version? It's then the responsibility of the user to input a string for the version. See https://github.com/puppetlabs/puppetlabs-apache/blob/master/manifests/mod/event.pp#L51 for example.